### DISCUSSION TITLE
gatsby-plugin-ts-config and Gatsby v4

### DISCUSSION THREAD

**Discussion by openscript** [Js-Brecht/gatsby-plugin-ts-config#52] [General]
I've updated a project with about 1000 pages to Gatsby v4, while using gatsby-plugin-ts-config 2.1.3. Everything seems to work well! I've also updated the starter I work on: https://github.com/openscript/gatsby-starter-dogmatism

What is your experience?

**Comment by Js-Brecht** [comment]
I haven't upgraded any sites to v4 yet, but I did get to play around with it while it was in Alpha.  Seems pretty cool.
